paper/paper.tex

```diff
@@ -50,11 +50,13 @@ \subsection{The enclose API}
 \end{figure}
 
 In \emph{\RE}, the \texttt{solver} is an instance of a struct that must be a subtype of \texttt{AbstractEnclosureAlgorithm}.
-If a user wants to add a new solver, they just have to add a new struct, say, \texttt{MyEnclosure} and implement the method \texttt{\_enclose}, which is automatically called by \texttt{enclose}.
+If a user wants to add a new solver, they just have to add a new struct, say, \texttt{MyEnclosure} and extend the method \texttt{enclose}, as the following code snippet demonstrates.
 %
 \begin{lstlisting}[language=Julia]
-_enclose(solver::MyEnclosure, f::Function,
-         D::Union{Interval, IntervalBox}; kwargs...)
+function enclose(f::Function, dom::Union{Interval,IntervalBox},
```

Comment:
or we can export our `Interval_or_intervalBox` ? (or whatever that's called?)

Reply:
I think it's fine to also not have the type annotation on the second argument

Reply:
I think having the type is informative, and being explicit with `Union{Interval,IntervalBox}` is easier to understand.
It also connects nicely to the text in the paper:

```latex
Note that \texttt{D} can be of type \texttt{Interval} for univariate ($n = 1$) functions
or of type \texttt{IntervalBox} for multivariate ($n > 1$) functions.
```

Reply:
I feel re-exporting `Interval` just for this is a bit of an overshoot, the proper way would be to type `IntervalArithmetic.Interval`.

Since in the example below we already do `import RangeEnclosure: enclose` maybe we can add a line
`using IntervalArithmetic: Interval`